chia/data_layer/s3_plugin_service.py

```diff
@@ -330,7 +330,7 @@ async def add_missing_files(self, request: web.Request) -> web.Response:
                         log.debug(f"skip {file_name} already in bucket")
                         continue
 
-                    with concurrent.futures.ThreadPoolExecutor() as pool:
+                    with concurrent.futures.ThreadPoolExecutor(thread_name_prefix="s3-upload-") as pool:
```

Comment:
may as well be unique from the upload group?

```suggestion
                    with concurrent.futures.ThreadPoolExecutor(thread_name_prefix="s3-missing-") as pool:
```